### DISCUSSION TITLE
High praise for JRUbuntu and can you help me with this small hurdle?

### DISCUSSION THREAD

**Discussion by Bahazzie** [Joshua-Riek/ubuntu-rockchip#612] [General]
RK3588 Ubuntu works fantastic so far. I have one small problem: I want to run my OPI5 via RealVNC without monitor. However as soon as my monitor plugs out everything stops working. I tried the trick that seems to work on Armbian by adding to ubuntuEnv the below line:

- extraargs=video-HDMI-A:1920x1080M@60

But it does not work. Any other ideas? I just want the OS to behave as if a 1080p monitor is connected, that's all.  

With the monitor not connected, it's not that I get a wrong resolution, I just get a totally unresponsive screeen, nothing works,

Thanks for any tips!

Bas

**Comment by Bahazzie** [comment]
As a side note: I have turned Wayland off, because for several reasons I want to use RealVNC and it works perfectly fine.  As long as monitor is connected.

**Comment by Joshua-Riek** [comment]
Please add your param to the `bootargs=` line in the file `/boot/firmware/ubuntuEnv.txt`. 

see: https://github.com/Joshua-Riek/ubuntu-rockchip/wiki/Kernel-Command-Line

**Comment by Bahazzie** [comment]
Hm no luck so far...I tried the below to /boot/firmware/ubuntuEnv.txt to the bootargs line

- extraargs=video-HDMI-A:1920x1080M@60

No luck. Then I tried to just add: video-HDMI-A:1920x1080M@60 to the bootargs line. Again no luck. 

Bas

**Reply by Joshua-Riek** [reply]
what is the output of the below command:
```
cat /proc/cmdline
```

**Comment by Bahazzie** [comment]
bas@bas-desktop:~$ cat /proc/cmdline
root=UUID=d6250cd5-a4ca-49e2-b0b8-0ba7bb333163 rootfstype=ext4 rootwait rw console=ttyS2,1500000 console=tty1 cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory swapaccount=1 systemd.unified_cgroup_hierarchy=0 quiet splash plymouth.ignore-serial-consoles extraargs=video-HDMI-A:1920x1080M@60

**Reply by Joshua-Riek** [reply]
remove the `extraargs=` text in `/boot/firmware/ubuntuEnv.txt`

**Comment by Bahazzie** [comment]
Then it says:
bas@bas-desktop:~$ cat /proc/cmdline
root=UUID=d6250cd5-a4ca-49e2-b0b8-0ba7bb333163 rootfstype=ext4 rootwait rw console=ttyS2,1500000 console=tty1 cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory swapaccount=1 systemd.unified_cgroup_hierarchy=0 quiet splash plymouth.ignore-serial-consoles video-HDMI-A:1920x1080M@60

Then after reboot and logging in via the actual keyboard/monitor, I disconnect the monitor. Next I connect via VNC from remote laptop and the VNC connection is accepted. Screen is shown, screen is unresponsive, and going from black to screen and back to black. As soon as I plug monitor in again VNC is back to normal.

Bas

**Reply by Joshua-Riek** [reply]
`extraargs=` is Armbian specific syntax and this OS comes with Panfork pre-installed for GPU drivers. Armbian does not have these drivers. If you do not want GPU drivers then uninstall them and VNC will work, this will not be fixed at any point as the Panfork project is dead.

**Reply by Bahazzie** [reply]
Thank you very much for taking the time to analyze this Joshua, it is much appreciated. I tried a couple of other things but no luck yet. There seems a hardware solution (HMDI hotplug, 7$) but I am not ready for that yet.

Bas

**Comment by hqnicolas** [comment]
@Bahazzie I have a bag with [this HDMI plugs](https://pt.aliexpress.com/item/1005002063364850.html?spm=a2g0o.productlist.main.1.c67f745dSyDkxN&algo_pvid=922de5d6-d339-4772-b789-c203609d0c23&utparam-url=scene%3Asearch%7Cquery_from%3A)
